cmd/server/cmd.go

```diff
@@ -18,6 +18,8 @@ import (
 	"io"
 	"time"
 
+	"github.com/streamnative/oxia/common/security"
```

Comment:
Run `make lint` will group `github.com/streamnative/oxia`'s import to same order.